packages/shared/prisma/seed.ts

```diff
@@ -521,6 +542,17 @@ function createObjects(
 
     scores.push(...traceScores);
 
+    if (i !== DEMO_USER_ID_NUMBER) {
+      const user = {
+        id: `user-${i}`,
+        email: `user-${Math.random()}@langfuse.com`,
+        name: `User ${i}`,
+        image: `https://avatar.iran.liara.run/public/${i}`,
```

Comment:
Chose some easy avatar generation page, please let me know how to ensure this url is secure & if you have strong feelings on not introducing this dependency. If in doubt, happy to remove! Intension for adding was only for marginal UI improvement.

Reply:
Adjusted, thank you 👍